gtests/net/mptcp/regressions/close_mpj_timeout2.pkt

```diff
@@ -0,0 +1,37 @@
+// connection initiated by packetdrill
+--tolerance_usecs=100000
+`../common/defaults.sh
+sysctl -q net.mptcp.close_timeout=1`
+
+// TODO keep using ADD_ADDR to keep test passing, until DSS values in packets 9 ~ 15 is understood
++0     `../common/multi-ep.sh -e 1 -m signal`
+
++0     socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0     setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
+
++0     bind(3, ..., ...) = 0
++0     listen(3, 1) = 0
++0.0     <  addr[caddr0] > addr[saddr0]  S   0:0(0)                     win 65535  <mss 1460, nop, wscale 7,  mpcapable v1 flags[flag_h] nokey>
++0.0     >                               S.  0:0(0)           ack 1                <mss 1460, nop, wscale 8,  mpcapable v1 flags[flag_h] key[skey]>
++0.0     <                                .  1:1(0)           ack 1     win 256                              <mpcapable v1 flags[flag_h] key[ckey=2, skey]>
++0     accept(3, ..., ...) = 4
+
+// the 4th ack, making the mptcp socket fully established, will trigger the add_addr
++0.0     <                                .  1:1(0)           ack 1     win 256                                         <dss dack4=1 nocs>
++0.0     >                                .  1:1(0)           ack 1                <add_address address_id=1 addr[saddr1] hmac=auto>
++0.0     <                                .  1:1(0)           ack 1     win 256    <add_address address_id=1 addr[saddr1] addr_echo,  dss dack4=1 nocs>
+
```

Comment:
here why don't you not exchange data to be in fully established mode?

Reply:
I'm unsure I read the above comment correctly. the msk socket will be fully established after line 20 (4th ack). We don't need to exchange additional data - and such case is covered by some other test-case.

Reply:
indeed, my bad!